docs/source/data_acquisition/SentinelTI.rst

```diff
@@ -0,0 +1,149 @@
+Microsoft Sentinel Threat Intelligence
+======================================
+
+MSTICPy integrates with Microsoft Sentinel's Threat Intelligence features, allowing users to list, create
+and update indicators in the Threat Intelligence Feature.
+https://learn.microsoft.com/azure/sentinel/work-with-threat-indicators
+
+Get Indicators
+--------------
+
+It is possible to return details of all indicators within a workspace.
+Whilst it is possible to access these indicator details via the indicator table in the Workspace, you can also interact
+with them via the Microsoft Sentinel APIs which are utilized in these functions.
+
+See :py:meth:`get_all_indicators <msticpy.context.azure_sentinel.MicrosoftSentinel.get_all_indicators>`
+
+.. code:: ipython3
+
+    azs.get_all_indicators()
```

Comment:
maybe sentinel.get_all_indicators (vs "azs")?

Reply:
Updated this here (and in all other Sentinel docs)